### DISCUSSION TITLE
Is there any way to "lock" times?

### DISCUSSION THREAD

**Discussion by remirez** [easepick/easepick#36] [General]
I'm currently working with a setup where I use the time-plugin as well as the lock-plugin. The lock feature works wonderfully for any given date, however I'm looking for more precision. As of now, if my minDate is January 1st at 10:59am, I can still select January 1st at 9:00am.

**Comment by wakirin** [comment]
The lock plugin does not currently support time.
You can try to implement it yourself.

For example (if I understand your case correctly): https://jsfiddle.net/waki/tj4h5wor/

**Reply by remirez** [reply]
Yes that's exactly what I'm looking for! Thank you!
